README.md

```diff
@@ -16,6 +16,26 @@ web-based playground hosted on Github](https://google.github.io/xtoproto).
 
 More details about how to use xtoproto will be added soon.
 
+## Design
+
+xtoproto uses example data to infer a both *protobuf definition*, a *source
+record spec*, and a *mapping definition*.
+
+* The *protobuf definition* can be
+represented in a `.proto` file with a custom `message` type for the record being
+inferred.
+* The *source record spec* is a specification for well-formed input data. For a
+  CSV-to-proto translation, this would be a schema for the CSV input: names and
+  types of columns.
+* The *mapping definition* defines how to convert from the source record spec to
```

Comment:
Let's add more details about the mapping definition, including code pointers.

Reply:
This isn't resolved yet.

Reply:
No, it's definitely resolved.

Reply:
It would be nice if when someone marked a thread "resolved" if that action were clearer in the trail comments. Disputes over whether something is resolved are common. Even more common is the need for the reviewer to check that something was actually resolved. This UI seems to bury the resolved comments.

Reply:
The UI is not showing it, but I have been alternating clicking "resolve conversation" and "unresolve conversation" in both of my accounts. Those actions are hidden.